R/extractManualRecode.R

```diff
@@ -1,10 +1,22 @@
 #' Get manual recodes
 #'
-#' @param recodedList A data frame with the automatically recoded values.+
+#' @param recodedList A data frame with the automatically recoded values.
 #' @param varName Character string for the column containing the automatically recoded values.
-#' @return Returns a data frame which only includes the values that have to be recoded manually. It can be saved to excel for manual recoding.
+#' @return Returns a data frame which only includes the values that have to be recoded manually. It can be saved to Excel for manual recoding.
+#'
+#' @examples
+#' # example data frame
+#' df <- data.frame(id = 1:4,
+#'              country = c("Germany", "Scotland", "Sweden" , "Egypt"),
+#'              capital = c("Berlin", "Edinburgh", NA, "Cairo"))
+#' # extract values to recode manually
+#' manual_recodes <- extractManualRecode(recodedList = df, varName = "capital")
+#' print(manual_recodes)
+#' # export to Excel, edit, import
+#' filePath <- paste0(tempdir(),"\\manual_recodes.xlsx")
```

Comment:
Also nice approach!
I found the following, which would make the `paste0()` obsolete:
```
temp <- tempfile(fileext = ".xlsx")
writexl::write_xlsx(manual_recodes, path = temp)
```